packages/fast-components-styles-msft/src/hypertext/index.ts

```diff
@@ -0,0 +1,16 @@
+import {IDesignSystem} from "../design-system";
+import {ComponentStyles} from "@microsoft/fast-jss-manager";
+import {IHypertextClassNameContract} from "@microsoft/fast-components-class-name-contracts";
+
+const styles: ComponentStyles<IHypertextClassNameContract, IDesignSystem> = {
```

Comment:
let me know if you, kham, and Jeff want to jump on a session to do one of these.